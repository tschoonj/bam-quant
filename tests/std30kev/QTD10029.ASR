File: C:\xrfdaten\2014KW30\chris\std30kev\std10029.spe
Start: 22.07.2014 10:07:02
Stop: 22.07.2014 10:08:42
BESSY: 297.9000145 298.0419233
IONI: 1.996519E-11 1.97403E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98124
Run: 100000
Dead: 2
ICR: 240
OCR: 0
Accumulated BESSY: 59708.9954493
Accumulated Ioni: 3.96482101E-09
N Accumulation: 200
Normfactor:  5.1408054e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
exp_x 112.720397949219 112.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  29   1      8.041      5913.        77.        .68
