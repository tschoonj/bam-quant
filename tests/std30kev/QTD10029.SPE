$DATA:
0 2000
23147
14897
6340
1788
362
54
5
0
0
0
0
0
0
0
0
0
0
13
11
15
15
10
14
11
10
10
8
10
7
13
12
12
7
7
10
13
10
9
7
7
6
6
8
5
9
11
7
8
8
11
6
10
8
8
10
11
7
5
7
10
6
6
8
11
6
4
8
4
4
7
5
1
10
9
6
7
5
4
7
6
5
6
8
3
2
6
8
6
5
9
10
6
6
6
6
4
7
3
6
6
5
3
9
13
11
8
3
6
6
5
6
7
7
6
7
4
5
3
9
4
9
8
10
7
2
6
8
7
7
11
5
5
9
1
8
7
8
9
8
9
7
10
8
10
9
2
9
7
3
9
7
6
4
6
7
6
6
3
7
7
8
8
10
7
4
4
5
9
9
5
5
7
4
2
3
11
11
5
5
6
3
5
4
3
9
3
2
3
3
5
7
3
7
7
3
9
5
5
6
0
1
3
2
6
4
1
4
3
5
3
6
2
0
2
3
7
3
7
5
5
2
3
3
2
3
2
4
0
1
2
1
7
5
2
3
2
1
3
1
4
7
1
1
3
5
3
1
4
0
1
6
2
3
1
0
2
1
0
2
2
1
3
3
3
3
4
2
4
3
1
2
5
1
2
1
1
1
5
3
0
2
2
1
3
4
3
4
3
1
2
3
1
2
1
3
1
1
4
3
0
2
1
1
0
1
4
2
4
3
5
3
4
5
5
6
5
7
9
4
6
1
6
1
0
1
2
1
1
2
3
1
3
1
2
1
1
2
1
0
3
3
2
2
1
2
3
2
3
3
4
8
5
4
4
3
1
2
2
1
2
5
6
2
2
3
3
1
2
2
5
4
2
2
0
4
3
1
2
4
1
3
4
1
6
1
4
7
9
16
20
41
86
128
226
325
456
590
662
682
659
617
486
359
245
147
93
53
34
14
10
8
1
3
5
5
1
5
3
6
10
9
3
4
4
9
12
8
10
16
20
18
25
42
69
124
161
238
344
466
565
604
651
606
538
446
336
228
150
87
51
28
6
4
1
2
1
0
0
3
0
1
0
0
1
0
0
0
0
1
0
0
2
2
1
0
1
1
3
1
1
1
2
1
1
1
0
0
0
1
1
1
1
1
0
2
1
1
0
2
4
0
1
1
1
0
1
0
0
0
3
0
2
0
1
0
0
1
1
2
0
3
2
0
2
0
0
1
3
2
0
0
1
1
0
2
0
0
0
2
4
0
1
1
1
1
1
1
0
0
2
0
1
4
0
1
2
1
1
2
3
3
0
1
2
2
1
2
2
3
2
1
3
2
1
3
1
3
4
1
1
2
3
1
2
3
2
3
2
2
1
3
0
1
0
2
1
1
5
3
0
3
4
0
2
0
0
2
1
3
3
2
0
2
4
3
1
1
1
3
0
0
4
2
1
2
1
9
2
3
3
1
2
2
2
2
2
0
3
2
4
4
1
4
3
1
0
1
1
1
3
1
2
1
2
3
1
3
0
1
3
5
5
6
5
2
3
2
1
3
1
3
4
5
3
2
5
4
3
3
1
1
5
2
6
3
3
3
3
7
1
3
4
5
1
9
4
3
3
4
3
1
4
4
4
1
2
2
3
3
2
2
6
8
2
2
3
3
3
3
3
4
8
4
2
7
5
4
5
4
7
4
6
6
2
4
2
1
6
0
9
2
5
5
7
6
2
7
4
3
2
6
6
2
2
4
3
3
4
0
7
6
5
11
7
8
0
8
6
8
10
7
26
21
32
29
53
54
76
78
79
73
69
62
80
57
39
34
30
16
15
9
7
11
4
5
4
5
4
4
2
5
4
1
7
2
9
7
6
2
4
3
4
1
4
3
3
3
3
7
5
4
4
4
7
4
3
5
1
3
3
3
2
0
11
8
2
2
3
3
10
3
2
3
8
7
1
2
1
3
5
4
3
7
5
7
2
8
4
6
7
2
4
2
5
3
7
7
6
3
8
8
8
10
12
17
15
15
21
14
7
10
14
8
10
2
5
6
4
4
1
4
3
4
6
6
7
2
4
7
7
5
4
4
4
7
6
4
4
1
2
5
5
4
3
2
5
1
5
1
5
3
3
8
4
2
4
2
5
6
7
8
1
3
6
2
2
3
4
0
1
2
1
2
3
6
8
2
2
5
6
2
4
3
5
3
6
7
8
7
3
4
0
2
2
6
1
2
5
5
5
3
3
7
2
1
3
4
6
3
6
3
4
4
1
6
3
5
4
7
3
2
3
2
4
5
0
5
4
1
2
4
4
3
1
5
2
2
6
6
3
3
2
3
1
1
3
6
0
3
0
3
3
7
9
5
4
4
3
3
3
4
3
4
4
3
4
4
1
5
8
1
2
5
6
1
2
3
3
4
3
4
1
3
2
5
4
0
2
5
4
6
2
1
3
3
6
5
2
2
0
2
2
3
4
1
6
4
5
6
5
3
2
7
4
5
8
9
6
10
5
12
11
8
11
3
8
10
7
7
8
5
4
3
7
6
2
5
2
3
0
1
0
2
2
6
2
5
1
3
9
4
0
2
3
2
2
2
4
3
2
2
3
2
2
2
2
5
4
4
2
3
3
4
1
1
2
3
2
4
4
5
2
3
3
2
1
2
2
5
2
2
1
5
2
3
3
4
2
1
2
1
3
0
1
2
5
0
2
2
1
2
2
1
2
1
2
1
3
3
0
5
2
2
4
2
4
1
1
3
4
3
4
2
2
2
3
2
5
2
1
3
2
3
2
2
2
1
2
0
2
1
2
2
0
2
5
1
1
4
5
9
7
10
12
19
18
24
30
30
21
27
20
24
28
27
35
36
30
33
41
34
35
29
37
25
20
23
20
18
17
13
9
4
7
5
6
2
1
2
5
0
3
3
2
2
2
2
2
1
1
2
2
2
0
0
1
1
2
2
1
2
2
1
2
4
2
0
2
3
2
1
3
2
1
2
4
3
4
2
3
0
0
0
0
0
1
1
3
2
1
1
0
0
2
0
1
1
2
2
0
0
3
2
2
1
1
2
1
3
2
3
4
2
3
2
3
1
2
6
5
3
0
4
2
1
1
2
4
3
1
2
5
3
5
1
7
1
3
1
1
1
2
2
4
2
1
2
3
0
2
2
2
3
3
1
2
0
1
1
8
1
2
0
2
3
2
1
2
2
3
5
4
8
7
7
3
9
17
10
14
9
10
15
8
7
14
4
7
5
8
5
3
3
4
3
0
1
6
3
3
1
0
5
6
2
2
4
6
4
4
4
6
5
5
1
5
7
3
1
3
5
4
3
5
4
6
2
3
6
2
2
1
4
1
4
4
6
4
2
6
5
9
5
2
1
4
5
3
4
3
4
2
7
2
3
3
6
3
10
5
4
5
5
1
3
3
3
6
3
5
7
4
1
6
5
5
3
3
6
2
4
7
7
5
9
5
11
10
8
8
5
7
5
4
8
4
4
5
7
10
6
8
6
9
7
9
4
6
8
10
8
8
11
7
9
4
12
12
8
9
13
7
7
6
8
15
11
10
15
11
11
7
17
12
16
14
9
11
20
16
15
6
8
14
6
11
19
14
13
20
12
17
24
16
16
18
17
15
19
22
27
22
17
27
12
21
24
23
20
21
24
16
29
19
25
20
23
32
28
42
19
30
35
31
21
32
37
30
23
28
21
29
28
30
35
28
27
31
32
29
28
36
34
27
29
36
32
27
27
36
27
32
29
31
33
32
25
35
19
27
21
21
34
33
23
22
26
21
26
23
31
20
37
26
25
20
24
22
27
19
18
18
24
27
18
36
35
24
29
18
19
20
27
23
27
21
17
18
22
26
19
16
20
23
21
12
14
22
21
16
25
17
18
14
21
12
21
16
23
17
13
17
18
21
22
17
30
15
37
30
26
29
28
41
40
31
46
52
48
52
56
67
59
59
54
56
67
49
60
65
56
55
55
45
59
53
54
40
55
46
33
45
27
32
30
32
27
17
26
25
14
10
13
16
12
11
9
6
6
2
3
2
2
0
1
0
2
0
0
0
2
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10029.spe
Start: 22.07.2014 10:07:02
Stop: 22.07.2014 10:08:42
BESSY: 297.9000145 298.0419233
IONI: 1.996519E-11 1.97403E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98124
Run: 100000
Dead: 2
ICR: 240
OCR: 0
Accumulated BESSY: 59708.9954493
Accumulated Ioni: 3.96482101E-09
N Accumulation: 200
Normfactor:  5.1408054e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
 
exp_x 112.720397949219 112.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.89
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  29   1      8.041      5913.        77.        .68
