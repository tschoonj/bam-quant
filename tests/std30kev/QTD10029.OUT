     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:47  บ   
     บ  Spectrum: QTD10029.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        3 iterations done    บ   
     บ  ChiSquare =     5.4   last change =   .04%    lambda= 1.E-05   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -40.7 ๑   100.0   บ     -40.7 ๑    12.6   บ   
     บ  GAIN (eV/ch)  บ     20.249 ๑   2.000   บ    20.249 ๑    .030   บ   
     บ det NOISE (eV) บ       93.6 ๑    40.0   บ      93.6 ๑    37.9   บ   
     บ  FANO factor   บ       .106 ๑    .050   บ      .106 ๑    .028   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cu-Ka                              5913. ๑     77.           บ   
     บ    KA1         8.048    .66084        3907. ๑     51.       .7  บ   
     บ              399.469    164.13          22.           9.57E-01  บ   
     บ    KA2         8.028    .33916        2005. ๑     27.       .5  บ   
     บ              398.481    163.99          22.           9.57E-01  บ   
     บ                                                                 บ   
     บ  2 Cu-Kb                              5727. ๑     76.           บ   
     บ    KB1         8.905   1.00000        5727. ๑     76.      1.0  บ   
     บ              441.792    169.92          25.           9.68E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
